async_sender/api.py

```diff
@@ -71,7 +74,8 @@ def __init__(
         self.port = port
         self.username = username
         self.password = password
-        self.use_tls = use_tls
+        self.use_tls = use_tls,
```

Comment:
Some tests was failed, I guess for this comma. Can you remove it, I will merge this PR after that.